polar-core/src/resource_block.rs

```diff
@@ -583,26 +640,22 @@ fn shorthand_rule_head_to_params(head: &Term, resource: &Term) -> Vec<Parameter>
     ]
 }
 
-// TODO(gj): better error message, e.g.:
-//               duplicate resource block declared: resource Org { ... } defined on line XX of file YY
-//                                                  previously defined on line AA of file BB
-fn check_for_duplicate_resource_blocks(blocks: &ResourceBlocks, resource: &Term) -> Result<()> {
-    if blocks.exists(resource) {
-        let msg = format!("Duplicate declaration of '{}' resource block.", resource);
-        let term = resource.clone();
-        return Err(ValidationError::ResourceBlock { msg, term });
-    }
-    Ok(())
-}
-
-fn check_that_shorthand_rule_heads_are_declared_locally(
+fn check_that_shorthand_rule_heads_are_declared_globally(
```

Comment:
This function was removed in [`8d414fb` (#1400)](https://github.com/osohq/oso/pull/1400/commits/8d414fbb087bd495e4bffb2010186cd608089e9a) b/c it was duplicative of the check in `ResourceBlocks::get_declaration_in_resource_block`

Reply:
✂️